src/utils/cliHelper.js

```diff
@@ -8,12 +8,33 @@ const git = require('git-state')
 const dirExist = dir => fs.existsSync(dir) && fs.statSync(dir).isDirectory()
 const fileExist = file => fs.statSync(file).isFile()
 
+const commitCheckParams = ['cat-file', 'commit']
+const childProcess = require('child_process')
+
 class CLIHelper {
   constructor(config) {
     this.config = config
     this.repoSetup = new RepoSetup(config)
   }
 
+  validateGitSha() {
+    const errors = []
+    ;['to', 'from'].forEach(field => {
+      const { status } = childProcess.spawnSync(
+        'git',
+        [...commitCheckParams, this.config[field]],
+        { cwd: this.config.repo }
+      )
+      if (status != 0) {
+        errors.push(`${field} ${this.config[field]} is not a valid commit`)
+      }
+    })
```

Comment:
**suggestion(_nit_):** maybe it could be written with a functional approach

```suggestion
;["to", "from"]
.map((field) => ({
  field,
  exist: childProcess.spawnSync(
    "git",
    [...commitCheckParams, this.config[field]],
    { cwd: this.config.repo }
  ),
}))
.filter((struct) => struct.exist.status !== 0)
.forEach((struct) =>
  errors.push(
    `${struct.field} ${this.config[struct.field]} is not a valid commit`
  )
);
```

Something like that